### DISCUSSION TITLE
[Feature] Mobile Improve Select albums UI

### DISCUSSION THREAD

**Discussion by bphenriques** [immich-app/immich#7134] [Feature Request]
### The feature

Hello,

Sorry if this is actually a duplicate ticket. Tried to search and I haven't find anything.

I have music stored in my Android phone with album pictures. When I use immich, it is hard for me to select the folders I want to backup as it shows an never-ending list of unsorted folders with a single file.

Usability suggestions/feedback:
1. Consistent sorting:
   1. Put "Recent" then Camara on the top-level as they are likely to be the first option. Perhaps put common social folders as  well on the top.
   2. Remaining folders sorted alfabetically (natural order). 
2. Add a scrollbar with alphabet hint (not sure if possible with the suggestion 1.i.).
3. Clarify what is "recent"? It seems that it includes music albums which is not what I want.
4. The UX to select/ignore faolders is weird. What is the difference between ignoring and just not-selecting a folder? I fell onto the trap of thinking that I was selecting a folder to backup after double-tapping. I tried to remedy the situation by clicking on the pill, only to end-up going back to scrolling through the list again and do a single tap.

Alternatively which is different, maybe users can instead manually _pick_ the folders on their phone rather than _selecting_ from a list.

Thank you for the project!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by jbtrystram** [comment]
Also, having the option to have an album created for each folder would be nice :)
